dropwizard-hibernate/src/main/java/io/dropwizard/hibernate/UnitOfWorkApplicationListener.java

```diff
@@ -84,7 +85,7 @@ else if (event.getType() == RequestEvent.Type.RESP_FILTERS_START) {
                         commitTransaction();
                     } catch (Exception e) {
                         rollbackTransaction();
-                        this.<RuntimeException>rethrow(e);
+                        throw new MappableException(e);
```

Comment:
Good point. So we could simply catch `RuntimeException` instead of `Exception` and re-throw after we rolled back the transaction. No need to wrap it in either `MappableException` or `RuntimeException`. This should make life easier for us. :smile: